pkg/apis/cluster/v1alpha1/machine_types.go

```diff
@@ -153,7 +166,6 @@ type MachineStatus struct {
 	// serialized/deserialized from this field.
 	// +optional
 	ProviderStatus *runtime.RawExtension `json:"providerStatus,omitempty"`
-
```

Comment:
Since ProviderStatus and Addresses are not directly related it would be nice to keep this newline here to more easily differentiate.

Reply:
Done, thanks!